### PR TITLE
Ports a bunch of brain damage lines from Bubberstation. This shit ain't nothing to me, man.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Ports a bunch of brain damage lines from Bubberstation. This shit ain't nothing to me, man.

## Why It's Good For The Game

chrome_d78i7PCd5Q

## Changelog
:cl:
spellcheck: This shit ain't nothing to me, man.
/:cl:

### CODE REVIEW

**Comment by Timonkeyn** [comment]
i dont think brain damage makes you form sentences perfectly

**Comment by tralezab** [comment]
👍 if it was converted into a pr better but brain damage lines are holy ground

**Review by MrMelbert — COMMENTED** [review]
This literally makes half the weight of the brain damage list the new lines, can we change that

Like we can make it to only 10% of the time it picks one of these lines easily

**Comment by carlarctg** [comment]
this should be from a drug like monkey energy or spray tan

**Comment by Kapu1178** [comment]
didnt you try to get the extremely mild sexual innuendos from valentine's day removed, but you're now pring "I'm on that Judge Judy pussy, snipe"?

**Comment by Zergspower** [comment]
image
bruh, the vday ones were so obscure too

**Comment by Iamgoofball** [comment]
it is now way rarer + pg-13

File: strings/traumas.json [hunk 1]
```diff
@@ -88,6 +89,95 @@
...
+	"We smokin' Sequoia banshee boogers.",
+	"We snortin' that good buffalo soldier tamarind Jordanian jimmies.",
+	"They must have amnesia, they forgot that I'm him.",
+	"That Burberry back woods pack hitting that pussy smell like a Hellcat V8.",
+	"We smokin' shit in a glass pipe, blowin' the lord's bubbles, yeah!",
+	"I'm sick in the head, I'm on them Broward County Tic-Tacs.",
+	"I'm on them Georgetown Gеronimos, I'm on them Nashville Nibblers.",
+	"I lеft my Margielas in the Benz truck, I'll have to stunt on them next time.",
+	"I don't give a fuck if I go blind, I don't need to see the price tag anyways.",
+	"I'm high on 12 Jason Bournes, lookin' to beat the cum out of a thick fresh oak.",
```
Comment (Kapu1178) [inline strings/traumas.json:149]: very pg13 of you

Comment (Iamgoofball) [reply strings/traumas.json:149]: whoops,  missed this one

**Comment by Sadboysuss** [comment]
these don't sound like something a brain damaged person would say, just weird lines imo, maybe I don't get the referrence but already existing lines like "GIMME ROBUX JUST ENOUGH FOR SHIRT AND PANTS" fit way more.

**Comment by honkpocket** [comment]
> these don't sound like something a brain damaged person would say, just weird lines imo, maybe I don't get the referrence but already existing lines like "GIMME ROBUX JUST ENOUGH FOR SHIRT AND PANTS" fit way more.

I'm not going to pretend like dracula flow is any better, but roblox meme from 2009 isnt funny at all. I mean, it was for 5 seconds when I heard it the first time in 2014 but its been a while.

**Comment by Sadboysuss** [comment]
> > these don't sound like something a brain damaged person would say, just weird lines imo, maybe I don't get the referrence but already existing lines like "GIMME ROBUX JUST ENOUGH FOR SHIRT AND PANTS" fit way more.
> 
> I'm not going to pretend like dracula flow is any better, but roblox meme from 2009 isnt funny at all. I mean, it was for 5 seconds when I heard it the first time in 2014 but its been a while.

Let's add skibidi toilet lines then

**Comment by honkpocket** [comment]
> Let's add skibidi toilet lines then

this is the opposite of the point i was trying to make

**Comment by optimumtact** [comment]
I dunno man... it's just not that funny without the context of the video.

**Comment by CCC23** [comment]
:( womp womp

**Pull request closed without merging**